style.css

```diff
@@ -1,12 +1,17 @@
-.container {
+* {
   box-sizing: border-box;
+}
+
+body {
+  height: 100%;
   margin: 0;
-  min-width: 375px;
+  width: fit-content;
 }
```

Comment:
- [ ] As requested by the previous code reviewer, your project has a horizontal scroll on both mobile and desktop screens as you can see from the video below so make sure you adjust the style of your project to remove the horizontal scroll. 

  [Screencast from 03-04-2023 01:00:20 PM.webm](https://user-images.githubusercontent.com/92303891/222904029-0fe72279-d7fa-411c-bf53-3901761dc951.webm)

[OPTIONAL] Please make sure that you set the `margin` of the page body to `0` to remove the white space around the body. :pray: You can do this :point_down: 
  ```css
  * {
    margin: 0;
  }
  ```

Reply:
Hi @jerryowusu; 
Can you please help I'm getting stuck for resolving that scroll in mobile version